src/server/v2.0/handler/usergroup.go

```diff
@@ -151,3 +152,25 @@ func (u *userGroupAPI) UpdateUserGroup(ctx context.Context, params operation.Upd
 	}
 	return operation.NewUpdateUserGroupOK()
 }
+
+func (u *userGroupAPI) SearchUserGroups(ctx context.Context, params operation.SearchUserGroupsParams) middleware.Responder {
```

Comment:
do we need pagenation for this API?

Reply:
Yes we do. We have cases where user's belong to 100's of groups. 

Having pagination would be a plus

Reply:
Pagenation added